stl/msbuild/stl_atomic_wait/msvcp_atomic_wait.settings.targets

```diff
@@ -51,9 +51,6 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     </PropertyGroup>
 
     <ItemGroup>
-        <CppPreprocess Include="$(MSBuildThisFileDirectory)\..\..\src\msvcp_atomic_wait.src">
-            <Defines>LIBRARYNAME=$(OutputName.ToUpper())</Defines>
-        </CppPreprocess>
         <DefFromI Include="$(IntermediateOutputDirectory)\msvcp_atomic_wait.i">

```

Comment:
Although I am the opposite of an MSBuild expert, it appears that `DefFromI` and `DllDefName` above should be removed. According to my understanding, the removed `CppPreprocess` logic was running `cl /P` to preprocess `msvcp_atomic_wait.src` and replace `LIBRARYNAME`. That emits `msvcp_atomic_wait.i`, which was then transformed by this `DefFromI` step into a `.def` file, with a name computed by `DllDefName`. That was then consumed by `DllDef`, which has now been changed to consume `stl/src/msvcp_atomic_wait.def` directly (no longer looking at `DllDefName`).

If this is right, then leaving `DefFromI` would break the build (`msvcp_atomic_wait.i` is no longer generated). Leaving `DllDefName` would be harmless dead code.

I'll validate this change internally.

Reply:
I went ahead and pushed this change, as it appears to be okay in the internal build (which is unfortunately failing for an unrelated reason).